source/elements/oneDNN/source/graph/data_model.rst

```diff
@@ -0,0 +1,58 @@
+.. SPDX-FileCopyrightText: 2020-2021 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+==========
+Data Model
+==========
+
+oneDNN Graph uses logical tensor to describe data type, shape, and layout.
+Besides 32-bit IEEE single-precision floating-point data type, oneDNN Graph can
+also support other data types. The shape contains multiple dimensions, and the
+total dimension and the size of the dimension could be set as unknown.
+
+oneDNN Graph uses the following enumeration to refer to data types it supports.
+Different operation may support inputs and outputs with different data types, so
+it's suggested to refer to the definition page of each operation.
+
+.. rst-class:: enumlist
+
+**enum** *dnnl::graph::logical_tensor::data_type*
+
+    undef
+        Undefined data type (used for empty logical tensor).
+    f16
+        `16-bit/half-precision floating point <https://en.wikipedia.org/wiki/Half-precision_floating-point_format>`__.
+    bf16
+        `non-standard 16-bit floating point with 7-bit mantissa <https://en.wikipedia.org/wiki/Bfloat16_floating-point_format>`__.
+    
+    f32
+        `32-bit/single-precision floating point <https://en.wikipedia.org/wiki/Single-precision_floating-point_format>`__.
+
+    s32
+        32-bit signed integer.
+
+    s8
+        8-bit signed integer.
+    
+    u8
+        8-bit unsigned integer.
+
+    boolean
+        boolean data type. The tensor element will be interpreted with C++ bool type.
+        Note that the size of C++ bool type is language implementation defined.
```

Comment:
Probably we don't need mention boolean here as we don't have related APIs and operations.

Reply:
Makes sense, I removed those until we get a stable functional implementation.